libcaja-private/caja-file-operations.c

```diff
@@ -1112,7 +1132,13 @@ do_run_simple_dialog (gpointer _data)
 			continue;
 		}
 
-		gtk_dialog_add_button (GTK_DIALOG (dialog), button_title, response_id);
+		if (button_title == CANCEL)
```

Comment:
Like it was in another PR, here strings are compared in improper way. However, I don't know if `g_strcmp0` could really be used for strings wrapped with `_( ... )` for translation.